jaraco/functools.py

```diff
@@ -422,9 +525,10 @@ def print_yielded(func):
     return functools.wraps(func)(print_results)
 
 
-def pass_none(func):
-    """
-    Wrap func so it's not called if its first param is None
+# TODO: Overload this as soon as https://github.com/python/mypy/issues/8881 is fixed
+# When T is None (Literal[None]), R becomes None too. Beware!
```

Comment:
[PEP8 recommends](https://peps.python.org/pep-0008/#maximum-line-length) reflowing comments and docstrings at 72 characters.

```suggestion
# TODO: Overload this type definition as soon as
# https://github.com/python/mypy/issues/8881 is fixed.
# When T is None (Literal[None]), R becomes None too. Beware!
```